### PR TITLE
add link to React

### LINKED ISSUE #9: Add React
https://facebook.github.io/react/index.html

[Contributing guide](https://facebook.github.io/react/contributing/how-to-contribute.html)
Issue label: [good first bug](https://github.com/facebook/react/labels/good%20first%20bug)

### PR DESCRIPTION
**Pull request by bpaysen**
closes #9

### CODE REVIEW

**Comment by erictleung** [comment]
Although this is a place to find ways to contribute to open source, I don't think it fits within the section of "Contributing to open source **in general**", because it is only towards helping one repository, rather than open source in general.

So either this addition isn't appropriate for this list or it should be put into another section. Those would be my two cents on the matter. Thanks for the PR though!

**Comment by bpaysen** [comment]
Thanks for the feedback Erictleung!

**Comment by erictleung** [comment]
@bpaysen ...butttt if this PR (https://github.com/FreeCodeCamp/how-to-contribute-to-open-source/pull/21) gets merged, I think your addition will best be under the new Projects section.

**Comment by nguyenduong8085** [comment]
Add link

File: README.md [hunk 1]
```diff
@@ -13,6 +13,7 @@ If you have questions or comments, please create an issue.
 - [YourFirstPR](https://twitter.com/yourfirstpr) - starter issues on GitHub that can be easily tackled by new contributors.
 - [Awesome-for-beginners](https://github.com/MunGell/awesome-for-beginners) - a GitHub repo that amasses projects with good bugs for new contributors, and applies labels to describe them.
```
Comment (nguyenduong8085) [inline README.md]: Github for email

Comment (nguyenduong8085) [inline README.md]: Thanks

Comment (nguyenduong8085) [inline README.md]: Github for nguyenduong8085

**Comment by nguyenduong8085** [comment]
@@ -13,6 +13,7 @@ If you have questions or comments, please create an issue.
 - [YourFirstPR](https://twitter.com/yourfirstpr) - starter issues on GitHub that can be easily tackled by new contributors.
 - [Awesome-for-beginners](https://github.com/MunGell/awesome-for-beginners) - a GitHub repo that amasses projects with good bugs for new contributors, and applies labels to describe them.

**Comment by erictleung** [comment]
@bpaysen hi there! Thanks for contributing! So we recently decided to create a separate file for projects and organizations to contribute to, which you can find [here](https://github.com/freeCodeCamp/how-to-contribute-to-open-source/blob/master/PROJECTS.md). Please rebase your commits and move your changes over to the separate file `PROJECTS.md`. If you have any questions for how to do that, let us know. Thanks!

**Comment by erictleung** [comment]
Closing as stale. @bpaysen please comment back if you're still interested in contributing. Happy coding!

**Pull request closed without merging**